README.md

```diff
@@ -30,26 +30,42 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The biggest difference between .forEach and .map is that .map creates a new array automatically while with .forEach you have to manually establish a new array.
+
 2. What is the difference between a function and a method?
 
+functions are actions that intake data, do something, and then return a result. Methods take functions and applythem to a given object (object, array, class, etc.).
+
 3. What is closure?
 
+Closure is the process of structuring a function's block of code in order to enable smaller functions to be run inside.
```

Comment:
and the smaller/nested function would have access to the outer function's variables.